### PR TITLE
docs: add details on platinum connectors

### PR DESCRIPTION
**Pull request by tayloramurphy**

### CODE REVIEW

File: docs/src/_cloud/connectors.md [hunk 1]
```diff
@@ -62,3 +62,29 @@ There are multiple ways to improve a connector's quality. Please [review this pa
...
+To learn more about which connectors are Platinum Certified, please [reach out to the team](https://meltano.com/contact/) or contact your account representative if you're a current Cloud customer.
+
+### Platinum Certified Connector - Guarantees
+
+Each Platinum Connector will have the following guarantees:
+
+* Built with the Meltano SDK
+* Open Source License
+* Semantic Versioning used in all releases
+* Daily integration testing
```
Comment (WillDaSilva) [inline docs/src/_cloud/connectors.md:80]: What will handle this? Do we have an issue open for this?

Comment (tayloramurphy) [reply docs/src/_cloud/connectors.md:80]: @WillDaSilva I was planning to bring this to the Singer Guild. Meltano Cloud is the solution here. We could have specific projects for these connectors running to any number of targets (duckdb at first or even snowflake) along with specific dbt models doing tests to validate the data. 

Adds a nice benefit for drinking our own Cloud champagne _and_ for regular testing of these first-party connectors.

File: docs/src/_cloud/connectors.md [hunk 2]
```diff
@@ -62,3 +62,29 @@ There are multiple ways to improve a connector's quality. Please [review this pa
...
+* Proactive API and changelog monitoring
```
Comment (tayloramurphy) [inline docs/src/_cloud/connectors.md:81]:
```suggestion
* Priority bug fixes
* Proactive API and changelog monitoring
```

File: docs/src/_cloud/connectors.md (hunk 1 above)
Comment (WillDaSilva) [reply docs/src/_cloud/connectors.md:80]: @tayloramurphy Do we have any issues open to track this? If we do, they may impact:
- https://github.com/meltano/meltano/issues/2566

**Review by edgarrmondragon — APPROVED** [review]

**Review by kgpayne — APPROVED** [review]

File: docs/src/_cloud/connectors.md (hunk 1 above)
Comment (tayloramurphy) [reply docs/src/_cloud/connectors.md:80]: I started https://github.com/meltano/internal-product/discussions/244

**Pull request merged**